### ISSUE TITLE
Destination image/alt text display does not work properly

### ISSUE THREAD

**Issue by jirassimok** [linuxboss182/SoftEng-2017#152]
This is just a formatting issue.

The image of the destination needs to be formatted to display at the right size and position, and the text that displays when there is no image needs to be centered.

**Comment by linuxboss182** [comment]
@ItsJackEMoore set CSS to center this, along with the other elements

**Comment by ItsJackEMoore** [comment]
Decided to wait for styling when FXML files are finalized

**Comment by ItsJackEMoore** [comment]
Based on the UI changes, is this issue still relevant?

**Comment by linuxboss182** [comment]
I don't think this is relevant anymore, closing issue
